pystac/validation/local_validator.py

```diff
@@ -111,7 +111,7 @@ def _validate_from_local(
 
     def _validator(self, stac_type: str, version: str) -> Draft7Validator:
         schema = _read_schema(f"stac-spec/v{version}/{stac_type}.json")
-        return Draft7Validator(schema, registry=self.registry)
+        return Draft7Validator(schema, registry=self.registry())
```

Comment:
I think I meant to make self.registry a property like in the `JsonSchemaSTACValidator` class

Reply:
Ah, should we change that?

Reply:
Eh. Doesn't super matter since this whole thing is deprecated.